frontend/public/components/utils/scroll-to-top-on-mount.tsx

```diff
@@ -13,3 +13,5 @@ export class ScrollToTopOnMount extends React.Component {
     return null;
   }
 }
+
+export const useScrollToTopOnMount = () => React.useEffect(() => document.getElementById('content-scrollable').scrollTo(0, 0), []);
```

Comment:
Refactored to TypeScript and added a [custom hook](https://reactjs.org/docs/hooks-custom.html) with same functionality (used in `CreateOperandForm` component).